docs/operation/upgrade.rst

```diff
@@ -0,0 +1,84 @@
+Upgrade Guide
+=============
+This section describes a reliable upgrade path for **MetalK8s** including
+all the components that make up the stack.
+
+.. important::
+
+    - Upgrading MetalK8s is a manual process and as a result,
+      the following outlined steps must be met to the later in other to achieve
+      a successful upgrade.
+    - The output of the below-provided commands solely depends on the particular
+      MetalK8s setup you have in place.
+    - **<destination_version>**
+      is a suffix which must be replaced with appropriate version number.
+
+      ::
+
+        #Get current version
+        cat /srv/scality/metalk8s*/product.txt | grep ^VERSION | cut -d'=' -f 2
+
+Supported Versions
+******************
+.. note::
+
+    MetalK8 supports upgrade **strictly** from one supported
+    minor version to another. For example;
+
+    - Upgrade from 2.0.x to 2.1.x
+
+    Please refer to the release notes for more information.
+
+Upgrade Pre-requisites
+**********************
+Prior to beginning the upgrade steps listed below, make sure to complete the
+pre-requisites listed in :doc:`/operation/preparation`.
+
+Upgrade Steps
+*************
+This section describes the logical steps specific **only** to Upgrading
+MetalK8s. Ensure that the pre-requisites above have been met before you make
+any step further.
+
+#. Upgrade the :term:`Bootstrap node`.
+
+    .. warning::
+        This command must be executed only on the
+        :term:`Bootstrap node`. Before proceeding, make sure that
+        this is verified.
+
+    .. code::
+
+       salt-call --local state.sls metalk8s.roles.bootstrap saltenv=metalk8s-<destination_version> pillar="{'metalk8s': {'endpoints': $(salt-call --out txt pillar.get metalk8s:endpoints | cut -c 8-)}}"
+
+#. From the :term:`Bootstrap node`, connect to the :term:`Salt Master`.
+
+    ::
+
+        crictl exec -it $(crictl ps -q --label io.kubernetes.container.name=salt-master) bash
+
+#. From the :term:`Salt Master`, launch the rolling upgrade.
```

Comment:
I updated the ticket yesterday to add a command to check what will be done. I think you should add it here